code/game/objects/items/toys.dm

```diff
@@ -1034,15 +1034,16 @@
 	desc = "A compact ball of snow. Good for throwing at people."
 	icon = 'icons/obj/toy.dmi'
 	icon_state = "snowball"
-	throwforce = 12 //pelt your enemies to death with lumps of snow
```

Comment:
keep this damage, add a `damtype = STAMINA` and remove the changes in `throw_impact`